backend/src/test/java/com/huellapositiva/integration/ResendEmailConfirmationActionShould.java

```diff
@@ -68,7 +72,11 @@ void update_hash_and_resend_email() {
         EmailConfirmation newEmailConfirmation = jpaEmailConfirmationRepository.findByEmail(DEFAULT_EMAIL)
                 .orElseThrow(() -> new UsernameNotFoundException("User with username: " + DEFAULT_EMAIL + " was not found."));
         String newHash = newEmailConfirmation.getHash();
-        assertThat(initialHash, is(not(newHash)));
+        Instant newCreationTimestamp = newEmailConfirmation.getCreatedOn();
```

Comment:
Creation date should never change, we should have a different field called updatedOn